librz/main/rizin.c

```diff
@@ -92,8 +92,8 @@ static int main_help(int line) {
 	if (line != 1) {
 		printf(
 			" --           run rizin without opening any file\n"
-			" -            same as 'rizin malloc://512'\n"
-			" =            read file from stdin (use -i and -c to run cmds)\n"
+			" -Pm          same as 'rizin malloc://512'\n"
```

Comment:
how was `-Pm` chosen? I think in the issue that was suggested for something else... this should probably be `=`, so one can easily open rizin with `rizin =`, to work on a malloc area.

Reply:
Yes, in issue it wasn't clear at all what this `-Pm` is about. It's better to just use `=` here.

Reply:
It seems that I suggested using `-Pm` (short for `--plugin m`) in https://github.com/rizinorg/rizin/issues/616#issuecomment-778616264 as an alternative for `=` (short for `malloc://512` after the changes in this PR).

Honestly, I think that adding the `--plugin` flag only makes sense if we think that other plugin shortcuts will be added in the future, but I really doubt that.

Because of this, I also think that it is better to simply use `=` to refer to `malloc://512` and forget about `-P` / `--plugin` for now.